draft-ietf-taps-interface.md

```diff
@@ -133,9 +133,11 @@ informative:
 --- abstract
 
 This document describes an abstract application programming interface, API, to the transport
-layer, following the Transport Services Architecture. It supports the
-asynchronous, atomic transmission of messages over transport protocols and
-network paths dynamically selected at runtime. It is intended to replace the
+layer that enables the selection of transport protocols and
+network paths dynamically at runtime. This API supports faster deployment
```

Comment:
I suggest /supports/enables/  ... Just because the word /supports/ is later used to describes what the API offers.

Reply:
```suggestion
network paths dynamically at runtime. This API enables faster deployment
```